docs/contributor/migration-guide.md

```diff
@@ -0,0 +1,85 @@
+# Migrating from Kyma CLI to `modulectl`
+
+This guide provides detailed instructions for migrating from the current Kyma CLI tool to the new `modulectl` tool.
+It covers all necessary changes and deprecations to ensure a smooth transition.
+
+## Overview
+
+`modulectl` is the successor of the module developer facing capabilities of Kyma CLI.
+It is already tailored for the update ModuleTemplate metadata as discussed in [ADR: Iteratively moving forward with module requirements and aligning responsibilities](https://github.com/kyma-project/lifecycle-manager/issues/1681).
```

Comment:
Did you mean tailored for updating ModuleTemplate metadata?

Reply:
I meant to say "tailored for the updated ModuleTemplate metadata"

Reply:
that was my second guess :) Thanks!